portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** A comment on a project. */
+public final class Comment {
```

Comment:
I actually only used a final because the walkthroughs had followed that convention. I don't see a super strong reason in favour of keeping this, so I could change...what do you think?

Reply:
Final is fine; I don't imagine that there's a reason to extend this class.